app/src/main/java/com/nearbynest/app/data/remote/model/PropertyTax.kt

```diff
@@ -0,0 +1,21 @@
+/**
+ * Copyright COPYRIGHT
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.nearbynest.app.data.remote.model
+
+data class PropertyTax(
+    val yearId: Int,
```

Comment:
Have you ever seen ID of the year? It can be either `year` or `id`.

Reply:
Yes, naming should have been better. Changed it to 'year'.